packages/docs/pages/Switch/SwitchPage.tsx

```diff
@@ -0,0 +1,34 @@
+import { Form, FormGroup, H0, H1, Switch } from '@bigcommerce/big-design';
+import React, { useState } from 'react';
+
+import { CodePreview } from '../../components';
+import { SwitchPropTable } from '../../PropTables';
+
+const SwitchPage = () => (
+  <>
+    <H0>Switch</H0>
+
+    <CodePreview>
```

Comment:
Same thing with the `TimePicker`, I would add some fluff text to get a quick synopsis.